GUIDELINES.md

```diff
@@ -0,0 +1,135 @@
+# sweetest test development guidelines
+
+After some time working with sweetest we came up to the conclusion that it leaves a lot of freedom to design tests as we want. But on the other side there is an increasing need for alignment. These guidelines are here for reaching an appropriate level of alignment. Feel free to challenge the current state and to contribute!
+
+## Content
+
+* [Goals](#goals)
+* [One-by-one steps](#one-by-one-steps)
+* [Principles](#principles)
+* [Links](#links)
+
+## Goals
```

Comment:
Nice overview!

Reply:
Agree, this makes some design decisions clearer!